sympy/stats/tests/test_continuous_rv.py

```diff
@@ -386,7 +386,7 @@ def test_beta():
 def test_beta_noncentral():
     a, b = symbols('a b', positive=True)
     c = Symbol('c', nonnegative=True)
-    _k = Symbol('k')
+    # _k = Symbol('k')
```

Comment:
Any reason to keep this at all?

Reply:
I guess it should go with the test below that currently uses str (but shouldn't really use str). Maybe I should just fix those tests...

Reply:
Okay I've fixed this test and another in the same file so that they use dummy_eq instead of a str-test.